cluster.yaml.example

```diff
@@ -17,7 +17,8 @@ releaseImage: quay.io/openshift-release-dev/ocp-release:4.2.0
 originReleasePrefix: quay.io/openshift-release-dev
 apiNodePort: 32323
 openVPNNodePort: 32324
-endpointPublishingStrategyScope: External
+endpointPublishingStrategy:
```

Comment:
This is not backwards compatible.  Are you intending to cherry-pick this to previous release?  I think you are.  If so, this must be backwards compatible so please leave `endpointPublishingStrategyScope` as-is.